app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/integration/NgVpnNetworkStack.kt

```diff
@@ -57,6 +58,7 @@ class NgVpnNetworkStack @Inject constructor(
     private val runtime: Runtime,
     private val appTrackerDetector: AppTrackerDetector,
     private val trackingProtectionAppsRepository: TrackingProtectionAppsRepository,
+    private val appTpFeatureConfig: AppTpFeatureConfig,
```

Comment:
This would then become 
```suggestion
    private val appTpLocalFeature: AppTpLocalFeature
```